code/modules/reagents/reagent_containers/glass/bottle/robot.dm

```diff
@@ -26,9 +22,4 @@
 	icon = 'icons/obj/chemical.dmi'
 	icon_state = "bottle-4"
 	reagent = "anti_toxin"
-
-/obj/item/weapon/reagent_containers/glass/bottle/robot/antitoxin/New()
-	..()
-	reagents.add_reagent("anti_toxin", 60)
-	update_icon()
-
+	prefill = list("anti_toxin" = 60)

```

Comment:
this prefill proc is so much less retarded than the old way and I like it

Reply:
You can thank me for that, and then thank Bluehaus for shamelessly stealing it ;D